plugins/connectrpc/dart/v0.1.0/buf.plugin.yaml

```diff
@@ -0,0 +1,12 @@
+version: v1
+name: buf.build/connectrpc/dart
+plugin_version: v0.1.0
+source_url: https://github.com/connectrpc/connect-dart
+integration_guide_url: https://connectrpc.com/docs/dart/getting-started
+description: Generates client stubs for connectrpc. Compatible with the gRPC, gRPC-Web, and Connect RPC protocols.
```

Comment:
Is this description accurate for the `connec-`dart implementation, it supports gRPC-Web?